samples/gcp-storage/README.md

```diff
@@ -105,6 +105,10 @@ So, the next step is just create a get request using postman, as the following i
 
 ![postman-response](./images/result-postman.png)
 
+When it create or modify a file in the bucket, it trigger the background function.
```

Comment:
Assuming the function doesn't create or modifies the file i'd change this to: 

"When a file gets created or modified in the bucket, the background function gets triggered"

So it is more clear that the function gets triggered when something happens in the bucket

Reply:
Done ec70c4127c3d03fe77b57d6f234f1e8ed39ac6b6